builtin/logical/ssh/util.go

```diff
@@ -215,6 +216,15 @@ func convertMapToStringValue(initial map[string]interface{}) map[string]string {
 	return result
 }
 
+func convertMapToIntValue(initial map[string]interface{}) map[string]int {
+	result := map[string]int{}
+	for key, value := range initial {
+		i, _ := value.(json.Number).Int64()
```

Comment:
This will panic if the input isn't a json.Number. I think this needs an error return param and if the input is bad, just error out to the user.

Reply:
Updated in 225443c to use `parseutil`